useful-scripts/misc-test-utils/src/utils/guid.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Guid } from './guid';
+
+describe('Guids', () => {
```

Comment:
We normally put the test files inside of `__tests__` directories adjacent to the associated file i.e.. `__tests__/Guid.test.ts`